recipes/sentencepiece/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "sentencepiece" %}
+{% set version = "0.1.86" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/google/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: a8744e5d2943112e24b69a7ec895d8ea26c41bb55464b78af9c396d10fb3e02a
+
+build:
+  number: 0
+  skip: True  # [py<36]
```

It's an option to skip macOS and Windows if you want to get this merged and then deal with macOS in the feedstock:
```suggestion
  skip: True  # [py<36 or win or osx]
```
I don't have time to help out or do a proper review at the moment, ping the staged recipes team once the CI is green and I'm sure someone will take a look.